README.md

```diff
@@ -1,23 +1,42 @@
 <p align="center">
-  <img src=".github/stochastic_logo_light.svg#gh-light-mode-only" width="600" alt="Stochastic.ai"/>
-  <img src=".github/stochastic_logo_dark.svg#gh-dark-mode-only" width="600" alt="Stochastic.ai"/>
+  <img src=".github/stochastic_logo_light.svg#gh-light-mode-only" width="250" alt="Stochastic.ai"/>
+  <img src=".github/stochastic_logo_dark.svg#gh-dark-mode-only" width="250" alt="Stochastic.ai"/>
 </p>
 
 <br>
 
-# ⚡️ Real-time inference for Stable Diffusion
+<!-- # ⚡️ Real-time inference for Stable Diffusion -->
 <!-- ![stochasticai_demo](.github/stochasticai_demo.gif) -->
 
-<p align="center">
+<!-- <p align="center">
  <img src=".github/stochasticai_demo.gif" width="600" alt="Stochastic.ai Demo"/>
-</p>
+</p> -->
+
+Welcome to `x-stable-diffusion` by Stochastic!
+
+This project is a compilation of acceleration techniques for the Stable Diffusion model to help you generate images faster and more efficiently, saving you both time and money.
+
+With example images and a comprehensive benchmark, you can easily choose the best technique for your needs. When you're ready to deploy, our CLI called `stochasticx` makes it easy to get started on your local machine. Try `x-stable-diffusion` and see the difference it can make for your image generation performance and cost savings.
 
-Stochastic provides this repository to perform fast real-time inference with `Stable Diffusion`. The goal of this repo is to collect and document as many optimization techniques for Stable Diffusion as possible. 
+## ✅ Stochastic
+
+Stochastic was founded with a vision to make deep learning optimization and deployment effortless. With our cloud platform, you can easily optimize and deploy your deep learning models with confidence, knowing that you are getting the best performance possible. Our platform automatically optimizes your models, benchmarking them on various evaluation metrics to ensure they are running at their peak.
+
+And when it comes time to deploy, Stochastic has you covered with auto-scaling accelerated inference for models like BLOOM 176B, Stable Diffusion, and GPT-J. Plus, our platform is cloud agnostic, supporting AWS, GCP, Azure, and Kubernetes clusters
+
+<p align="center">
+ <img src=".github/stochastic_x_dashboard.jpeg" width="600" alt="Stochastic X Dashboard"/>
+</p>
 
-Currently this repository includes 4 optimization techiques with more in the pipeline. Feel free to open a PR to submit a new optimization technique to the folder.
+For fully-managed solution hosted on Stochastic [Sign up →](https://www.stochastic.ai/)
```

Comment:
Should this link redirect to https://app.stochastic.ai/signup ?

Reply:
I think that would be better right? I will make this change.